.storybook/preview.js

```diff
@@ -32,5 +25,3 @@ addParameters({
 		showPanel: false,
 	},
 });
-
-configure(loadStories, module);
```

Comment:
In Storybook v6, `preview.js` is executed for each story, so I moved relevant parts of `config.js` to this file.